scripts/benchmarks.py

```diff
@@ -109,6 +109,17 @@ def accept(seed, testdata, value):
     return accept
 
 
+def lower_bound(seed, testdata, value):
+    rnd = random.Random(seed)
+    for b in testdata.buffer:
+        c = rnd.randint(0, 255)
+        if c < b:
+            return True
+        if c > b:
+            return False
+    return True
```

Comment:
Also able to read the code, but not getting *why* it would be called or where `value` comes into it.  Internals docstring here would be good 😄